src/optimizer_interface.jl

```diff
@@ -808,29 +808,52 @@ end
         model::GenericModel;
         allow_local::Bool = true,
         allow_almost::Bool = false,
+        allow_time_limit::Bool = false,
```

Comment:
I would prefer a list of accepted status instead of more keywords

Reply:
`allow_status = ()`, and I would suggest a tuple since it is non-allocating. But accepting any collection that work with `in` should be fine